test/preview/components/classifiers/test_file_classifier.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from haystack.preview.components.classifiers.file_classifier import FileTypeClassifier
+from test.preview.components.base import BaseTestComponent
+from test.conftest import preview_samples_path
+
+
+class TestFileClassifier(BaseTestComponent):
```

Comment:
This is a thing I realized is missing in many other cv2 components already: let's test che component's serialization. So far I think only `LocalWhisperTranscriber` does it. 

See how it's done in this very small test: https://github.com/deepset-ai/haystack/blob/bb7af3827d4822b2cd5151c8f6d76570c9d027bf/test/preview/components/audio/test_whisper_local.py#L16-L21